### PR TITLE
Add issue template to make it easier to submit bugs

### PR DESCRIPTION
**Pull request by etagwerker**
Hi folks,

I noticed we don't have an issue template in this project so I decided to add one.

This will give some structure to the process of submitting a bug to the project.

Thoughts? 

Thanks, 
Ernesto

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[BUG] "
+labels: 'bug'
+assignees: ''
```
Comment (bronzdoc) [inline .github/ISSUE_TEMPLATE/bug_report.md:6]: should the `assignees` be to the `black bunny brigade` team?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[BUG] "
```
Comment (bronzdoc) [inline .github/ISSUE_TEMPLATE/bug_report.md:4]: What if its a feature request?

Comment (etagwerker) [reply .github/ISSUE_TEMPLATE/bug_report.md:4]: @bronzdoc There is already a template for feature requests in this project. 👍 

When submitting a new issue, they will be given a choice to select one of these options: 

- Bug Report 
- Feature Request 

See: https://github.com/fastruby/skunk/blob/main/.github/ISSUE_TEMPLATE/feature_request.md

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+_Delete any information that is not relevant._
+
+If you are unable to reproduce the bug, add the **Non-Reproducible** tag and describe the steps you followed leading to the bug to the best of your recollection.
+
+**Screenshots and Videos**
+
+If the issue has an effect in the frontend, include any relevant screenshots and videos here.
+
+**Logs**
```
[the remark this answers is not in the document]
Comment (etagwerker) [reply .github/ISSUE_TEMPLATE/bug_report.md:60]: @bronzdoc I think it works better if we keep both actual/expected behavior sections brief. 

It is always useful to see "I expected it to do X." but "It actually did Y."

**Comment by etagwerker** [comment]
@bronzdoc I just updated the template based on your feedback. Thanks! 👍

**Review by bronzdoc — APPROVED** [review]

**Pull request merged**